src/resources/Resource.ts

```diff
@@ -1,15 +1,16 @@
-/* @flow */
-
 export default class Resource<TContext> {
   context: TContext;
 
-  // eslint-disable-next-line no-unused-vars
-  constructor(context: TContext, config: any) {
+  constructor(context: TContext) {
     this.context = context;
   }
 
-  // eslint-disable-next-line no-unused-vars
-  get(id: string): ?Object | Promise<?Object> {
+  get(
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
shouldn't need the pragma if it's underlined? Also maybe this should be an abstract method